content/client-server-api/modules/push.md

```diff
@@ -520,6 +520,33 @@ Definition:
 }
 ```
 
+**`.m.rule.room.server_acl`**
```

Comment:
Should this note somewhere that it was changed in 1.4?

Reply:
bother! I had one job...

Reply:
I'd probably pin this one on the reviewer than the author.

Reply:
https://github.com/matrix-org/matrix-spec/pull/1201